src/locale/translations/fr.json

```diff
@@ -153,7 +153,19 @@
         "HowToIsolateCTA": "Comment vous isoler"
       },
       "RegionCovered": {
-        "Body2": "Vous êtes à risque d’être infecté."
+        "Body2": "Vous êtes à risque d’être infecté.",
+        "ON": {
+          "RegionalGuidance": {
+            "CTA": "Go to COVID-19 Test Results Website",
+            "URL": "https://covid19results.ehealthontario.ca:4443/"
+          }
+        },
+        "NL": {
+          "RegionalGuidance": {
+            "CTA": "Find out what to do next",
+            "URL": "https://www.gov.nl.ca/covid-19/exposed/"
```

Comment:
Should be:
```
"CTA": "Découvrez si vous devriez vous faire tester",
"URL": "https://www.811healthline.ca/fr/autoevaluation-pour-la-covid-19/"
```
[ref](https://github.com/cds-snc/covid-alert-app/pull/1067/files#diff-423da31a5a4545a5e4df7702aadcf34dR401-R402)